src/message.ts

```diff
@@ -466,22 +467,19 @@ export class Message implements Sayable {
             name = mentionName + '@' + name
             nameList.push(name.slice(0, -1)) // get rid of the `@` at beginning
           })
-        return nameList
+        mentionList = mentionList.concat(nameList || [])
```

Comment:
Do not modify `mentionList` inside `multipleAt` function.

Do not do any side effect inside a `map` function, because it is a bad practice, decrease readability of the code, make other people confusing.